ts/mixpanel.ts

```diff
@@ -42,6 +41,14 @@ const setupMixpanel = async (mp: MixpanelInstance) => {
   await mp.identify(DeviceInfo.getUniqueId());
 };
 
+export const optOut = async () => {
```

Comment:
I'd rather name it `terminate` since it seems to revert what `init` does (at least within this module).
The opt-out semantics represents a specific use case.

Reply:
You're right, changed in 8eca449 in order to have a common semantic!